src/js/views/replyTweet.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Tweet from './tweet.js';
+import ReplyListener from './replyListener.js';
+import PostTweetUI from './postTweet.js';
+
+class ReplyTweet extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      display: 'none',
```

Comment:
let's not use inline styles in the state, keep the concept you are using the 'display' but use css